libs/datamodel/connectors/datamodel-connector/src/helper.rs

```diff
@@ -28,3 +28,17 @@ pub fn parse_u32_arguments(args: Vec<String>) -> Result<Vec<u32>, ConnectorError
         Ok(res.iter().map(|arg| *arg.as_ref().unwrap() as u32).collect_vec())
     }
 }
+
+pub fn arg_vec_from_opt(input: Option<u32>) -> Vec<u32> {
+    match input {
+        Some(arg) => vec![arg],
+        None => vec![],
+    }
+}
```

Comment:
nitpick (no need to change anything): this could be written more concisely as `input.into_iter().collect()` or `input.map(Vec::new)`.

Reply:
Done